### PR TITLE
Define jobcentre registers in alpha

### PR DESCRIPTION
**Pull request by trvllngmn**
registry files for checking.

### CODE REVIEW

File: data/alpha/field/local-authority.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: curie
+field: local-authority
+phase: alpha
+register: ''
+text: 'A local authority'
```
Comment (Jamie-Atkinson) [inline data/alpha/field/local-authority.yaml]: update to 
'A local authority in England, Scotland and Northern Ireland or a principal local authority in Wales.'

File: data/alpha/field/jobcenter-group.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: jobcentre-group
+phase: alpha
+register: 'jobcentre-group'
+text: 'A jobcentre office group'
```
Comment (Jamie-Atkinson) [inline data/alpha/field/jobcenter-group.yaml]: does not read right, maybe a grouping of jobcentre offices?

File: data/alpha/field/local-authority.yaml (hunk 1 above)
Comment (Jamie-Atkinson) [inline data/alpha/field/local-authority.yaml]: update to 
'A local authority in England, Scotland and Northern Ireland or a principal local authority in Wales.'

Comment (nacnudus) [reply data/alpha/field/local-authority.yaml]: Also rename to `local-authorities` and have `cardinality: "n"`

Comment (nacnudus) [reply data/alpha/field/local-authority.yaml]: Also rename to `local-authorities` and have `cardinality: 'n'`

File: data/alpha/register/jobcentre-district.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+fields:
+- jobcentre-district
+- name
+- local-authority
```
Comment (nacnudus) [inline data/alpha/register/jobcentre-district.yaml]: Rename to `local-authorities`

Comment (trvllngmn) [reply data/alpha/register/jobcentre-district.yaml]: ammended

File: data/alpha/field/local-authority.yaml (hunk 1 above)
Comment (trvllngmn) [reply data/alpha/field/local-authority.yaml]: ammended

File: data/alpha/field/jobcenter-group.yaml (hunk 2 above)
Comment (trvllngmn) [reply data/alpha/field/jobcenter-group.yaml]: ammended

**Comment by trvllngmn** [comment]
@Jamie-Atkinson @nacnudus  Changes have been made. Please review.

**Pull request merged**